src/include/func_specific/rng_seed_stream.hpp

```diff
@@ -5,7 +5,168 @@
+764129, 618740, 582752, 692507, 789955, 785851, 112601, 380578, 896709, 549581, 965817, 162355, 338288, 502799, 447091, 144368, 163130, 717741, 150842, 376320, 241741, 177798, 584469, 620725, 812482,
```

Comment:
Looks like the new values are not matching previous. Is this intentional

Reply:
This one is intentional, the random seeds should be high enough to increase uniformity in randomization, but should be lower than the max value of the type. Reduced the size to avoid an out of bound issue.